calibrators/marker_radar_lidar_calibrator/include/marker_radar_lidar_calibrator/marker_radar_lidar_calibrator.hpp

```diff
@@ -119,19 +132,21 @@ class ExtrinsicReflectorBasedCalibrator : public rclcpp::Node
     const std::vector<std::pair<Eigen::Vector3d, Eigen::Vector3d>> & matches,
     builtin_interfaces::msg::Time & time);
 
-  std::tuple<pcl::PointCloud<PointType>::Ptr, pcl::PointCloud<PointType>::Ptr, double, double>
-  getPointsSetAndDelta();
+  std::tuple<pcl::PointCloud<PointType>::Ptr, pcl::PointCloud<PointType>::Ptr> getPointsSet();
+  std::tuple<double, double> getDelta(std::vector<Track> converged_tracks, bool is_crossval);
+
   std::pair<double, double> computeCalibrationError(
     const Eigen::Isometry3d & radar_to_lidar_isometry);
-  void estimateTransformation(
-    pcl::PointCloud<PointType>::Ptr lidar_points_pcs,
-    pcl::PointCloud<PointType>::Ptr radar_points_rcs, double delta_cos_sum, double delta_sin_sum);
+  void estimateTransformation();
+  void calculateCalibrationError(Eigen::Isometry3d calibrated_radar_to_lidar_transformation);
+  void crossValEvaluation();
   void findCombinations(
     int n, int k, std::vector<int> & curr, int first_num,
     std::vector<std::vector<int>> & combinations);
-  void crossValEvaluation(
-    pcl::PointCloud<PointType>::Ptr lidar_points_pcs,
-    pcl::PointCloud<PointType>::Ptr radar_points_rcs);
+  void selectCombinations(
+    int tracks_size, int num_of_samples, std::vector<std::vector<int>> & combinations);
+  void doEvaluation(std::vector<std::vector<int>> & combinations, int num_of_samples);
```

Comment:
doEvaluation -> evaluate ?

Reply:
In context: evaluateCombinations?

Reply:
Thanks, fixed in https://github.com/tier4/CalibrationTools/pull/180/commits/2d0accd9bddda43bf71d4b1e15e6ef9d1be049bc